package.json

```diff
@@ -76,7 +77,7 @@
     "clean": "rm -rf dist release",
     "dist": "export GULP_ENV=\"production\" && npm run clean && npm run test && npm run build",
     "shrinkwrap": "npm-shrinkwrap --dev",
-    "test": "mocha --ui exports --reporter src/test/reporter/reporterSwitch --require babel-core/register --require babel-polyfill src/test/**/*.test.* src/test/*.test.*",
+    "test": "mocha --ui exports --reporter src/test/reporter/reporterSwitch --require babel-core/register --require babel-polyfill --require src/test/environment/jsdom  src/test/**/*.test.* src/test/*.test.*",
```

Comment:
This means we're always spawning a DOM even if we don't use it. When we get to https://github.com/mesosphere/marathon/issues/2928 it might make sense to even split out the 2 different testing approaches with 2 different scripts, so that for unit tests we skip this. What do you think?